posthog/tasks/test/test_usage_report.py

```diff
@@ -368,8 +378,7 @@ def _test_usage_report(self) -> List[dict]:
                     "plugins_enabled": {"Installed and enabled": 1},
                     "instance_tag": "none",
                     "event_count_lifetime": 55,
-                    "event_count_in_period": 22,
-                    # TODO: enhanced_persons: modify this test so that there are fewer of these events than the base
+                    "event_count_in_period": 23,
                     "enhanced_persons_event_count_in_period": 22,
```

Comment:
Test is failing for this right now, the test thinks it should be 23, but since the event I added above is propertyless, it should actually be 22 still.